examples/llms/RAG/static_evaluation_dataset.csv

```diff
@@ -0,0 +1,57 @@
+question,source,retrieved_doc_ids
+What is the purpose of the MLflow Model Registry?,['model-registry.html'],"['model-registry.html', 'introduction/index.html', 'introduction/index.html', 'deep-learning/index.html']"
```

Comment:
This is going to become pretty obsolete as we continue to refactor content over this quarter. How are we planning on updating this example?